### DISCUSSION TITLE
UI wireframes/sketches

### DISCUSSION THREAD

**Discussion by uo282892** [Arquisoft/lomap_en3a#48] [Ideas]
This discussion will serve as the place to present the sketches of the different parts of the user interface.

**Comment by uo282892** [comment]
# UI Main Page
![main_page](https://user-images.githubusercontent.com/98897888/224809150-94ce8973-d9d7-4b1b-b033-9fc720cedf14.png)

**Comment by GuillermoDylan** [comment]
# Friends Page

![image](https://user-images.githubusercontent.com/98907563/224311722-6cf230a7-f6bd-4aff-930e-c2c9c3d9c073.jpeg)

**Comment by Davidmc07** [comment]
# Home page


![untitled_page](https://user-images.githubusercontent.com/98897872/224511427-1b22d22e-75f7-4c2a-93e7-3ee593e205f9.png)

**Comment by Davidmc07** [comment]
# Home page

![ui2](https://user-images.githubusercontent.com/98897872/224795174-87e5ce31-0b82-48e2-b46e-78b86c6c3b03.PNG)

**Reply by UO283428** [reply]
The colors of this one are the best in my opinion.

**Reply by Pelayo-Reguera** [reply]
I like the shapes but the colors are horrible. I think that even the page of a funeral home is more colorful hahahahahaha

**Reply by GuillermoDylan** [reply]
I like the design, but the color palette seems to be very dark, maybe some tones of purple would be great.

**Reply by Pelayo-Reguera** [reply]
Or even blue like the palette from the Ivan's design

**Reply by uo282892** [reply]
As Guillermo says, the design is promising but the colors have no personality. From the colour palettes we have seen so far, I would choose either the one with different tones of blue, or another with different tones of purple and magenta (like the current login page).

**Comment by UO283428** [comment]
The two wireframes of David are really great I think is what we are looking for.

**Comment by GuillermoDylan** [comment]
#  **UI Redesign Proposal and Filter for the Map**
Hi guys, I have made some wireframes for the application, taking into consideration the [last approach that was made for the main page](https://github.com/Arquisoft/lomap_en3a/discussions/48#discussioncomment-5294932) and tried to redefine it a little bit, just to give it some color and try if it works out, I have made 2 different versions, with different palettes, but I'm also opened to any new color suggestions. Please let me know what you think.

![Main Page](https://user-images.githubusercontent.com/98907563/228647504-31132c5c-e6c5-4b73-85dd-6b6690828818.jpg)

![Main page (Second palette)](https://user-images.githubusercontent.com/98907563/228648184-e3546f52-dc27-488f-a09b-caf1aa1f6839.jpg)

## **Friends List**
I  have also planned a redesign for the Friends view, this is tied to a short discussion that took place during one of the meetings, during which an approach to show the friends image together with their information on the friend's list was made.

![Friends List](https://user-images.githubusercontent.com/98907563/228648485-fd47b9d8-e429-4916-bfe0-beee524fcf07.jpg)

Small note: if I am able to, I will also upload later a version with the color palette used in https://github.com/Arquisoft/lomap_en3a/discussions/48#discussioncomment-5294932.

**Reply by UO283428** [reply]
They are really good, nice work!! I really like the second one, I think the contrast is better than the one we have right now.
Can you share the color code of the two options? I think we can try some combinations of the first one with a purple like the one from Solid.

**Reply by GuillermoDylan** [reply]
Yes of course! The color codes are the following ones:

For the first palette:
> HEX: 451964 (Dark purple)
> HEX: D4B3EC (Light purple)

For the second one:
> HEX: 002E66 (Dark blue)
> HEX: B2CCEB (Light blue)

All of the wireframes share a background greyish color, which in hexadecimal: DEDEDE.

**Comment by uo282892** [comment]
I like very much the different tones of blues you used, they make a nice contrast and they fit very well.

However, the idea of having purple, apart from the fact that I am not a big fan of it, is that linking our application with solid aesthetics may look as if our LoMap does not have its own identity. What do you think?

**Comment by GuillermoDylan** [comment]
# Sketches for comments and reviews on the place information

![Overview](https://user-images.githubusercontent.com/98907563/230581733-df979885-f5ac-4d70-8157-3586016cb0c6.jpeg)

![Reviews](https://user-images.githubusercontent.com/98907563/230581692-6a5a706f-e791-4d54-8cd8-dba0edaee422.jpeg)

**Comment by GuillermoDylan** [comment]
# Suggestions for new page views?

#### Hi guys, I have here some basic sketches for the new page views we are set to implement, but the thing is that as I was thinking on how to display everything I found myself without really knowing what to display or how, so I would appreciate your opinion on this. I mainly want to know exactly what we are presenting to the user in the Groups page and how you would like it to he shown, I mean in terms of tables, lists, etc. 
![image](https://user-images.githubusercontent.com/98907563/233784208-76c751dd-cf23-40d6-b0d4-8aac7a65b5c2.jpeg)
![image](https://user-images.githubusercontent.com/98907563/233784214-a032144c-933b-43b8-b00b-d8204abaf530.jpeg)

**Reply by Davidmc07** [reply]
Yes, something like that. We can start by displaying the name, description and table of maps in the group view. It may be better not to add an icon and wait until everything else is implemented before displaying the member list, mainly because we would have to deal with updating it each time a user leaves the group
